packages/inter-protocol/test/auction/test-auctionBook.js

```diff
@@ -1,19 +1,19 @@
 import { test } from '@agoric/zoe/tools/prepare-test-env-ava.js';
 
 import { AmountMath } from '@agoric/ertp';
+import { eventLoopIteration } from '@agoric/notifier/tools/testSupports.js';
+import { buildManualTimer } from '@agoric/swingset-vat/tools/manual-timer.js';
 import { makeScalarBigMapStore } from '@agoric/vat-data';
-import { setupZCFTest } from '@agoric/zoe/test/unitTests/zcf/setupZcfTest.js';
+import { makeBoard } from '@agoric/vats/src/lib-board.js';
 import {
   makeRatio,
   makeRatioFromAmounts,
+  prepareRecorderKitMakers,
 } from '@agoric/zoe/src/contractSupport/index.js';
 import { makeOffer } from '@agoric/zoe/test/unitTests/makeOffer.js';
-import { buildManualTimer } from '@agoric/swingset-vat/tools/manual-timer.js';
-import { makeManualPriceAuthority } from '@agoric/zoe/tools/manualPriceAuthority.js';
-import { eventLoopIteration } from '@agoric/notifier/tools/testSupports.js';
 import { setup } from '@agoric/zoe/test/unitTests/setupBasicMints.js';
-import { makePublishKit } from '@agoric/notifier';
-import { makeBoard } from '@agoric/vats/src/lib-board.js';
```

Comment:
`makeBoard` is no longer used? Oh... I see it moved up. I wonder what's missing in our toolkit that results in these spurious diffs.

Reply:
https://github.com/Agoric/agoric-sdk/issues/7403